integrations/docker/images/chip-build/Dockerfile

```diff
@@ -178,6 +178,14 @@ RUN set -x \
 
 # Install a known ZAP release
 # Only keep the cli version, since `zap` is 143MB and not usable (UI)
+#
+# If you change this, make sure you also change (once used)
+#   - scripts/tools/zap/zap_execution.py
```

Comment:
What needs to be changed in zap_execution.py?

Reply:
Updated the comment: I created a script that will do a version update in all files that contain a zap version.